bridge/README.md

```diff
@@ -16,6 +17,14 @@
 - **Minimum SDK**: 28
 - **Target SDK**: 34
 
+## Building
+
+To build an aar for inclusion in consumer applications, run:
```

Comment:
`aar` should be `AAR` anywhere we refer to it.

Reply:
Cool 👍

b650813527fc3cd7e866fb1841d0fa24cd0c1145